internal/install/command.go

```diff
@@ -73,4 +81,6 @@ func init() {
 	Command.Flags().BoolVarP(&skipIntegrations, "skipIntegrations", "r", false, "skips installation of recommended New Relic integrations")
 	Command.Flags().BoolVarP(&skipLoggingInstall, "skipLoggingInstall", "l", false, "skips installation of New Relic Logging")
 	Command.Flags().BoolVarP(&testMode, "testMode", "t", false, "fakes operations for UX testing")
+	Command.Flags().BoolVar(&verbose, "verbose", false, "increase log level verbosity")
+	Command.Flags().BoolVar(&trace, "trace", false, "trace level logging")
```

Comment:
Consider short flag `vv`?

Reply:
I did and then read https://github.com/spf13/cobra/issues/679